qube/create/templates/common_files/{{ cookiecutter.commonName }}/.github/workflows/generate-reports.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow will build a package using Maven and then publish it to
+#  qbic-repo.qbic.uni-tuebingen.de packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Generate reports
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - development
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install git
+        run: sudo apt-get install git
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+          settings-path: {% raw %}${{ github.workspace }}{% endraw %} 
+
+      - name: Setup Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7' # Version range or exact version of a Python version to use, using SemVer's version range syntax
+          architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pygithub
+
+      - name: Load local Maven repository cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: {% raw %}${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}{% endraw %}
+          restore-keys: |
+              {% raw %}${{ runner.os }}-maven-{% endraw %}
+
+      - name: Maven generate reports
+        run: mvn --activate-profiles !development-build,!release-build --settings .github.settings.xml site
+
+      - name: Set up git
+        run: |
+            git config --global user.email "support@qbic.zendesk.com"
+            git config --global user.name "JohnnyQ5"
+
+      - name: Publish reports
+        run: |
+          python --version
```

Comment:
This should be reported by GA anyways, so it can be safely removed if you want to.

Reply:
nice spot, that was some left-over from the debugging